go/test/endtoend/messaging/message_test.go

```diff
@@ -30,14 +30,15 @@ import (
 
 	"vitess.io/vitess/go/mysql"
 	"vitess.io/vitess/go/sqltypes"
+	"vitess.io/vitess/go/vt/log"
+	"vitess.io/vitess/go/vt/vtgate/vtgateconn"
+
+	querypb "vitess.io/vitess/go/vt/proto/query"
+	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
+
```

Comment:
Seems strange to break these up into separate blocks if they are all for internal vitess imports?

Reply:
in most other files we tend to separate imports from the generated protobufs into their own blocks

Reply:
We seem very inconsistent about it though? Especially here where the protobuf ones split up multiple vitess import blocks and isn't entirely separate on their own? Anyway, just seems like churn for something we're don't seem to have standardized. And if we should, can we trick something like `goimports` into enforcing a consistent style? (I don't really even care that much about the style really).